test/integration/targets/filter_urls_jinja26/aliases

```diff
@@ -0,0 +1,3 @@
+shippable/posix/group2
+skip/python2.6  # filters are controller only, and we no longer support Python 2.6 on the controller
+skip/aix
```

Comment:
This new test should be embedded into the existing `filter_urls` test, since the name of the test is important. For plugin tests, the target name should be in the format: `{plugin_type}_{plugin_name}`

To do that, first convert the existing test to a script-based. You can see how that can be done without having to move the existing files in the test here: 

https://github.com/ansible/ansible/blob/devel/test/integration/targets/filter_core/runme.sh

Reply:
So to be clear: Run the existing test, then set up the venv and run this test after that?

Reply:
Yes